packages/tools/generator-pie-component/src/app/index.ts

```diff
@@ -48,6 +48,8 @@ export default class extends Generator {
     }
 
     async end () {
-        this.log(chalk`Your component has been created at ${this.props.componentPath}`);
```

Comment:
Not sure if it was just a Windows thing, but for me this was logging as:
```
Your component has been created at , packages/components/pie-test/
```
which looked a bit messy...